addons/website_blog/static/src/scss/website_blog.scss

```diff
@@ -99,12 +98,8 @@ $o-wblog-title-sizes-variants: (
         background-color: rgba(black, 0.005);
     }
 
-    .o_wblog_read {
-        max-width: $o-wblog-read-width;
-    }
-
     .o_wblog_read_with_sidebar {
-        max-width: $o-wblog-read-container-sidebar-width;
+        max-width: map-get($container-max-widths, md) + $o-wblog-sidebar-width;
```

Comment:
@sdegueldre 720 + 280 = 1000 now that we use container md width... and in fact, I tested and the layout with the sidebar is not centered anymore by just changing the class in the xml (since now you add margin by using the o_container_small class). I let you adapt the PR since more changes are needed than what I thought.

Reply:
As far as the 1000px is concerned, that was kind of the point: so that increasing the small container size wouldn't squish the side bar.

I removed the additional margin where applicable (in some places, I removed an mx-auto since it's covered by the container), and double checked that all affected layouts are now properly centered.